db/seeds.rb

```
@@ -5,3 +5,23 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+Image.create([{ url: 'https://picsum.photos/id/1/200/300' },
```

Usually good to prefer `create!` over `create` since it won't silently fail.